app/assets/javascripts/angular/controllers/static_pages.js.erb

```diff
@@ -0,0 +1,57 @@
+app.controller('StaticPagesCtrl', ['$scope', '$http', function($scope, $http){
+
+  $scope.book = {};
+  $scope.form = {};
+
+  $scope.form.create = function(){
+    var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
+
+    var data = {
+      book: $scope.book
+    };
+
+    $http.post(url + '/books', data).success(function(response){
+      console.log(response);
+    });
+  };
+
+  $scope.apiSearch = function(keyEvent, value) {
+    if (keyEvent.which === 13) {
```

Comment:
Are you trying to run this function on enter in a form? If so, you can do ng-submit.